packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -1,262 +1,262 @@
 [
   {
     "chunkShortName": "accordion",
-    "size": 4547,
-    "gzipSize": 1966
+    "size": 4550,
+    "gzipSize": 1967
   },
   {
     "chunkShortName": "banner",
-    "size": 5217,
-    "gzipSize": 2149
+    "size": 5212,
+    "gzipSize": 2147
   },
   {
     "chunkShortName": "button",
-    "size": 7179,
-    "gzipSize": 2883
+    "size": 7173,
+    "gzipSize": 2888
   },
   {
     "chunkShortName": "button-group",
-    "size": 1039,
-    "gzipSize": 588
+    "size": 1038,
+    "gzipSize": 587
   },
   {
     "chunkShortName": "button-pure",
-    "size": 7940,
-    "gzipSize": 3283
+    "size": 7934,
+    "gzipSize": 3280
   },
   {
     "chunkShortName": "button-tile",
-    "size": 6871,
+    "size": 6867,
     "gzipSize": 2893
   },
   {
     "chunkShortName": "carousel",
-    "size": 44126,
-    "gzipSize": 17756
+    "size": 44118,
+    "gzipSize": 17760
   },
   {
     "chunkShortName": "checkbox-wrapper",
-    "size": 8630,
-    "gzipSize": 3556
+    "size": 9890,
```

Comment:
About +1KB for each wrapper except for select-wrapper. Can this be right?

Reply:
Checkbox and Radio-Button Wrapper now have duplicated styling regarding the base positioning which was previously abstracted. But to have every component the same setup and abstraction it's duplicated now.